GOVERNANCE.md

```diff
@@ -2,6 +2,10 @@
 
 The Node.js Web Team (@nodejs/web) is a team in the Node.js Project that is composed by a set of subteams. Each containing specific responsibilities and goals.
 
+### TSC Oversight 
+
+Any website change that expresss a position about a global event or group of people requires explicit [TSC](https://github.com/nodejs/TSC/blob/main/TSC-Charter.md#section-4-responsibilities-of-the-tsc) approval.
```

Comment:
Can you also write down what would be considered TSC approval? Adding it to the TSC agenda and getting consensus there? Or, enough thumbs up? I believe we need better instructions here to failsafe this 👀

Reply:
I think pinging tsc is enough, if there are blockers then they will be discussed

Reply:
I think pinging the TSC is a pre-requisite. But what would he a TSC approval? No objections? One thumbs up? And for how long should we wait for the feedback of the TSC?

Reply:
I think the process should be "sending an email to tsc@iojs.org and receive no objections and at least one approval in seven days". I don't think the TSC meetings would be a good place to discuss this, it's probably better to have this sort of discussions in private (I could easily imagine that TSC members might not be in a position where they can express publicly political opinions)

Reply:
Im not sure how I feel about the TSC making such actions behind the scenes (through a private email) whereas the Website Team or the author of the contribution gets publicly exposed for idealising a patch that might contain a political statement.

I understand the want of anonymity but I believe it makes sense for at least one member of the TSC publicly express the approval. Otherwise it's just me saying "ah yes the TSC approved it" ...

Reply:
TSC approval is a documented process. The TSC follows the consensus seeking process and if it can't built consensus the TSC votes.

Reply:
Is this what Antoine commented, the official TSC approval process? 

Well, there's then very little I can do with that.

Reply:
On a related note, we have precedence for authoring a commit from the project (https://github.com/nodejs/nodejs.org/commit/a87f468982dde3e5931f4c3a2ee7c67cc6253e67), so no individual needed to take "the blame", although I'm not sure if that's what you're asking.
I think if some TSC members want to express publicly their approval, that's certainly fine, but we should not require it. If the website team is not comfortable moving forward if there are no public approvals, that's certainly fine, but I don't think it needs to be in the policy. Does that make sense?

Reply:
That's fair! Thanks for explaining, Antoine!